bedrock/privacy/templates/privacy/notices/mozilla-vpn.html

```diff
@@ -8,4 +8,4 @@
 
 {% set body_id = "mozilla-vpn" %}
 
-{% block article_header_logo %}{{ static('img/logos/vpn/logo-with-wordmark.svg') }}{% endblock %}
```

Comment:
**suggestion (blocking)**: let's delete this outdated `img/logos/vpn/logo-with-wordmark.svg` file. I don't see any other instances of use now, you caught them all 🥅